static/css/jira-configuration.css

```diff
@@ -199,6 +199,15 @@
   top: -7.5em;
 }
 
+/*
+The tooltip for the first two rows are hidden due to the `overflow: auto` of the parent div,
+so changing the position of the tooltips for the first two rows to bottom
+ */
+.jiraConfiguration__table__row:nth-of-type(1) .jiraConfiguration__syncWarningMessage,
```

Comment:
What does this change have to do with the ghe loading screen? PRs shoudl stay focused on a specific change. This should be in a separate PR with a description of why this change is needed + screenshot.

Reply:
This is related to a small UI issue that was discovered today. There were no tickets and seeing how small it was, decided to do the fix here. 
Discussed here: https://atlassian.slack.com/archives/C01PERFAUHZ/p1655252266197379
Should we do this separately in a new ticket then?

Reply:
We can let it slide this time but we generally keep PRs focused on a specific change, even if the additional change is tiny.

On the css here though: we can simplify this by using the nth-child selector and setting a range.

Reply:
Used `nth-child` with range